Switch Transcript ID and name column

msmanoj opened the pull request:
## Description

- Switched transcript ID and name columns
- Removed bolding of names

## Views affected
- http://ves-hx2-75.ebi.ac.uk:42229/Homo_sapiens/Gene/Summary?g=ENSG00000139618;r=13:32315086-32400268
- http://ves-hx2-75.ebi.ac.uk:42229/Homo_sapiens/Transcript/Summary?db=core;g=ENSG00000139618;r=13:32315086-32400268;t=ENST00000544455

## Related JIRA Issues

- https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6331
- https://www.ebi.ac.uk/panda/jira/browse/ENSWEB-6332

ens-st3 reviewed (commented):
I can't see any changes from live. Is your sandbox on the right branch ?

msmanoj commented:
> I can't see any changes from live. Is your sandbox on the right branch ?

@ens-st3 -  sorry,  I changed the branch on my sandbox. I have fixed it now. Please check.

ens-st3 reviewed (approved):
Looks good on a sandbox, and code is fine
